### PR TITLE
Add Typescript bindings for minijinja

### PR DESCRIPTION
**Pull request by virajmehta**

### CODE REVIEW

File: dashboard/package.json [hunk 1]
```diff
@@ -8,15 +8,18 @@
     "dev": "remix vite:dev",
     "lint": "eslint --ignore-path .gitignore --cache --cache-location ./node_modules/.cache/eslint .",
     "start": "remix-serve ./build/server/index.js",
+    "test": "vitest",
```
Comment (GabrielBianconi) [inline dashboard/package.json:11]: Same as other PR

File: dashboard/utils/minijinja/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# WASM Bindings for Minijinja
+
+This crate provides a WASM bindings for the Minijinja templating library.
+
+## Testing
+
+Run `wasm-pack test --node` from this directory.
```
Comment (GabrielBianconi) [inline dashboard/utils/minijinja/README.md:7]: Needs to go in CI

**Review by GabrielBianconi — CHANGES_REQUESTED** [review]
```
$  npm test -- --run



> test
> vitest --run


 RUN  v2.1.4 /Users/gabrielbianconi/Development/tensorzero/tmp/tensorzero/dashboard

 ❯ utils/minijinja.test.ts (0)

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯ Failed Suites 1 ⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯

 FAIL  utils/minijinja.test.ts [ utils/minijinja.test.ts ]
Error: Failed to load url ./minijinja/pkg (resolved id: ./minijinja/pkg) in /Users/gabrielbianconi/Development/tensorzero/tmp/tensorzero/dashboard/utils/minijinja.test.ts. Does the file exist?
 ❯ loadAndTransform node_modules/vite/dist/node/chunks/dep-BWSbWtLw.js:51920:17

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯[1/1]⎯

 Test Files  1 failed (1)
      Tests  no tests
   Start at  14:11:04
   Duration  240ms (transform 20ms, setup 0ms, collect 0ms, tests 0ms, environment 0ms, prepare 65ms)
```

Failed to reproduce, called the above after `cargo build`. Needs instructions (incl. `cargo install wasm-pack`).

Needs CI.

File: dashboard/utils/minijinja/Cargo.toml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+[dependencies]
+wasm-bindgen = "0.2.62"
+minijinja = { version = "2.1.0", features = ["loader"] }
+
+# The `console_error_panic_hook` crate provides better debugging of panics by
+# logging them with `console.error`. This is great for development, but requires
+# all the `std::fmt` and `std::panicking` infrastructure, so isn't great for
+# code size when deploying.
+console_error_panic_hook = { version = "0.1.6", optional = true }
```
[the remark this answers is not in the document]
Comment (virajmehta) [reply dashboard/utils/minijinja/Cargo.toml:23]: I made the default in the crate not use this feature so it should not be compiled by default

**Review by GabrielBianconi — APPROVED** [review]

**Pull request merged**